README.md

```diff
@@ -124,22 +99,21 @@ The builder image is responsible for building the iRODS HTTP API package. Before
 docker build -t irods-http-api-builder -f irods_builder.Dockerfile .
 ```
 
-With the builder image in hand, all that's left is to get the source code for the GenQuery2 project and HTTP API project. The builder image is designed to compile code sitting on your machine. This is important because it gives you the ability to build any fork or branch of the projects.
+With the builder image in hand, all that's left is to build the iRODS HTTP API project. The builder image is designed to compile code sitting on your machine. This is important because it gives you the ability to build any version of the project derived from this branch.
```

Comment:
I don't quite understand the last sentence:
> This is important because it gives you the ability to build any version of the project derived from this branch.

I think the advantage of being able to build from source code on the host filesystem is that you don't have to "sync" it anywhere (i.e. push to a remote repo in GitHub) in order to build it in the container(s). Being able to checkout a specific commit-ish to build can be done without the source code being on the host filesystem. Maybe I've misunderstood what this is saying.

Reply:
This is primarily about the removal of support for GenQuery2. If there are users who need GenQuery2 for iRODS 4.2.x, they can't use builder images derived from this commit.

Reply:
Can we just say that instead? The sentence as written does not seem related to GenQuery2 or iRODS 4.2.x at all.

Reply:
Sure. Will reword that.

Reply:
Adjusted wording on L102 and at the beginning of the `Build` section.

Reply:
Will likely link the GenQuery2 API plugin repo so people know what we're referring to.